src/components/Menu/index.tsx

```diff
@@ -311,7 +311,7 @@ export default function Menu() {
 
 interface NewMenuProps {
   flyoutAlignment?: FlyoutAlignment
-  ToggleUI?: React.FunctionComponent
+  ToggleUI?: React.FunctionComponent<React.PropsWithChildren<unknown>>
```

Comment:
Import these directly to avoid importing all of `React`.

Reply:
done